packages/sql/src/transforms/scales.ts

```diff
@@ -93,16 +133,16 @@ const scales = {
   utc: scaleTime
 };
 
-export function scaleTransform(options) {
+export function scaleTransform<T>(options: ScaleOptions): ScaleDescriptor<T> | null {
```

Comment:
Can this method really return null? At least from the types it seems like we always have a scale assuming that options.type is correctly typed. I found typescript works best when types are kept tight.

Reply:
Updated code to throw an error if an invalid scale type is provided. Also refactored the core package to use the scale types exported by the sql package.